### DISCUSSION TITLE
AutoTrain Custom Model Choice

### DISCUSSION THREAD

**Discussion by vizhukova** [huggingface/autotrain-advanced#561] [Q&A] [answered]
<img width="848" alt="Screenshot 2024-03-28 at 13 46 55" src="https://github.com/huggingface/autotrain-advanced/assets/13242508/7b2801ec-6e5c-46d9-89d7-72dd0d8478d9">

I see that in the https://huggingface.co/docs/autotrain/model_choice#autotrain-model-choice has opportunity to choose _Model Choice: AutoTrain_ or _HuggingFace Hub_  but not in stable "autotrain-advanced" package nor in _main_ version, installed from the source, I see it. Also I saw that features in lots of guide videos on youtube and still can not find it in GUI version or in github's one.
Was that feature has been removed? Or does it in development and only panning to be added?
If it was moved - why, was it too heavy for the envs?
Is there any expectation to bring that back?
Is there any opportunity to use that through the any paid version or via specific version of git repo?

**Comment by abhishekkrthakur** [comment, marked as answer]
two ways:
in spaces you can pass : /?custom_models=mistralai%2FMixtral-8x7B-Instruct-v0.1
OR
in spaces or locally, you can pass the model name in AUTOTRAIN_CUSTOM_MODELS  env var

**Comment by vizhukova** [comment]
So should it be something like that? (I used same config and data from my previous SFT and just added that emv var - after that my space all the time pausing after push "Start" button and in logs no errors)
<img width="677" alt="Screenshot 2024-03-28 at 17 49 14" src="https://github.com/huggingface/autotrain-advanced/assets/13242508/d901d017-584d-4ca8-acd7-52d6be9ac71b">

same as /?custom_models=mistralai%2FMixtral-8x7B-Instruct-v0.1 approach

**Comment by abhishekkrthakur** [comment]
sure. you can remove the new line. also, grok wont work :)
